src/app/suspect/page.tsx

```
@@ -1,103 +1,94 @@
+                            <h2
+                                className="text-2xl font-bold text-primary mb-4 font-top-secret"
+                            >
+                                {t('suspect.languages')}
+                            </h2>
+                            <div className="bg-gray-100 p-4 rounded-lg">
+                                {languages?.map(language => (
+                                    <LanguageLevels
+                                        language={language.language}
+                                        level={language.level}
+                                    />
+                                ))}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)